app/views/catalog/_index_recent_items.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="card">
```

Comment:
So the single item partial is called `index` but the collection partial is called `index`? That is confusing :(

Reply:
I agree. It was how Spotlight had it set up for auto-looking up partials. Since we're using less of that now I can try to avoid it.